src/mlpack/bindings/python/mlpack/io_util.hpp

```diff
@@ -12,7 +12,7 @@
  */
 #ifndef MLPACK_BINDINGS_PYTHON_CYTHON_IO_UTIL_HPP
 #define MLPACK_BINDINGS_PYTHON_CYTHON_IO_UTIL_HPP
-
+// TODO: Change this.
```

Comment:
What do you mean to change here?  We don't have to fix it yet, I'm just curious. :+1:

Reply:
oops, forgot to remove this TODO here, will do that in a commit